### ISSUE TITLE
Icons have .image extension instead of .png

### ISSUE THREAD

**Issue by rquinio** [asciidoctor/asciidoctor-maven-plugin#406]
When upgrading to asciidoctor-maven-plugin 2.0.0-RC.1 I've noticed a bug with using maven-site-plugin and attribute `<icons>image</icons>`.
The generated `<img>` src URIs for admonitions and callouts have ".image" extension instead of ".png":

`NOTE: Note`
is rendred as:
```html
<div class="admonitionblock note">
<table>
<tbody><tr>
<td class="icon">
<img src="./images/icons/note.image" alt="Note">
</td>
<td class="content">
Note
</td>
</tr>
</tbody></table>
</div>
```
With 1.5.7.1 it was `<img src="./images/icons/note.png" alt="Note">`

Full configuration: https://github.com/rquinio/asciidoctor-maven-demo/blob/de9491c586debef52949d93bb31c67ff406ab688/pom.xml#L182-L240

**Comment by abelsromero** [comment]
Not setting png as default seems an issue to me, do you want to report upstream to https://github.com/asciidoctor/asciidoctor?
I could trace it to this change in core https://github.com/asciidoctor/asciidoctor/issues/2953 introduced in asciidoctor-2.0.0.rc.1.

In the meantime, since v2.0.0, you can use just `<icons>png</icons>` or you can use `<icontype>png</icontype> with `<icons>image</icons>` witch is compatible with previous versions.

**Comment by rquinio** [comment]
Thanks for the tips !
I've doubled checked the doc https://github.com/asciidoctor/asciidoctor.org/blob/master/docs/_includes/icons.adoc and it says the <icons /> should have empty value (defaulting to png) - not sure why I used <icons>image</icons> in the first place, looks more like bad usage on my end...

**Comment by abelsromero** [comment]
I do think you found something, to me the behavior is not consistent or the documentation is misleading. Let's see what Dan thinks.

thanks for reporting :+1:

**Comment by mojavelinux** [comment]
This was a bug in Asciidoctor core which was fixed in 2.0.10. See https://github.com/asciidoctor/asciidoctor/issues/3308
